### PR TITLE
Creating a Readme for Repo to Star and Follow

### PR DESCRIPTION
**Pull request by bastosmichael**
Getting feedback for the Readme on the public sdk-code-gen repository.

[Preview Link](https://github.com/liblaber/sdk-code-gen/blob/049a663838393a5b1257b3b8af6217511697e2b1/README.md)

### CODE REVIEW

**Review by rcvrgs — COMMENTED** [review]
Can you add the preview link to the description? For easy access.

**Comment by bastosmichael** [comment]
Done, good call

**Comment by bastosmichael** [comment]
Screen Shot 2022-06-01 at 10 55 57 AM
Website URL already in the Repo's About section

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+> liblab generate-sdk --api="openapi.yaml"
+Generating SDK for Java...done ✓
+Generating SDK for Python...done ✓
+Generating SDK for JavaScript...done ✓
+Generating docs for developer portal...done ✓
+```
+
+[Join Our Community](https://discord.gg/F8aECHbRkV)
+
+Follow and Star this Repository to get the latest changes and updates as we airdrop our Beta MVP in the near future.
```
Comment (rcvrgs) [inline README.md:15]: I re-read this. Thinking we don't need to explicitly mention "MVP". I suggest updating it to "Beta launch".

Comment (bastosmichael) [reply README.md]: There is already a link back to liblab.com in the About section of the Repo and updated all of the above.

**Comment by bastosmichael** [comment]
https://github.com/liblaber/sdk-code-gen/blob/1e7efec4580b9641b1f039f7b1abcce106ca57ad/README.md Updated Preview

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
-# sdk-code-gen
-LibLab.com SDK Code Generator 
+# SDK Code Generator (Coming Soon)
+
+Please follow and star this project to get updates prior to our Beta Launch.
```
Comment (sagivo) [inline README.md:3]: Add coming soon

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+> liblab generate-sdk --api="openapi.yaml"
+Generating SDK for Java...done ✓
+Generating SDK for Python...done ✓
+Generating SDK for JavaScript...done ✓
+Generating docs for developer portal...done ✓
+```
+
+Instantly generate SDKs in multiple languages for your API service.
+
+[![Join Our Discord](https://user-images.githubusercontent.com/1518708/171464936-408b2120-449f-4003-b9d4-877deb4f14f8.png)](https://discord.gg/F8aECHbRkV)
```
[the remark this answers is not in the document]
Comment (bastosmichael) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (bastosmichael) [reply README.md:3]: It has (Coming Soon) in the title

**Pull request merged**